src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/controllers/WorkflowController.scala

```diff
@@ -106,18 +109,28 @@ class WorkflowController @Inject()(workflowService: WorkflowService) {
     workflowService.runWorkflowJobs(workflowId, jobIdsWrapper.jobIds).toJava.toCompletableFuture
   }
 
-  @GetMapping(path = Array("/workflow/export"))
-  def exportWorkflow(@RequestParam id: Long): CompletableFuture[ResponseEntity[ByteArrayResource]] = {
-    workflowService.exportWorkflow(id).map { workflowExport =>
-      val resource = new ByteArrayResource(
-        ObjectMapperSingleton.getObjectMapper.writerWithDefaultPrettyPrinter.writeValueAsBytes(workflowExport)
-      )
-
-      ResponseEntity.ok()
-        .contentType(MediaType.parseMediaType("application/json"))
-        .header(HttpHeaders.CONTENT_DISPOSITION, s"attachment; filename=${workflowExport.workflowJoined.name}.json")
-        .body(resource)
-    }.toJava.toCompletableFuture
+  @GetMapping(path = Array("/workflows/export"))
+  def exportWorkflows(@RequestParam jobIds: Array[Int]): CompletableFuture[ResponseEntity[ByteArrayResource]] = {
```

Comment:
I would create two endpoints, one for single workflow and one for multiple workflows.
Here is one function which does two things. 
But leave it as it is.

Reply:
I could extract the parts into two functions. I think two endpoints aren't necessary. If you would call `/workflows/export/42` you would get a json wrapped in a zip file, while `workflow/export/42` would return only the json. I think it doesn't make sense to have two ways of exporting.

Reply:
OK